packages/ebayui-core/src/components/ebay-phone-input/component.ts

```diff
@@ -15,14 +15,14 @@ export interface PhoneInputEvent {
     countryCode?: string;
 }
 
-interface PhoneInputInput extends Omit<ListboxButton, `on${string}`> {
+interface PhoneInputInput extends Omit<Marko.HTML.Span, `on${string}`> {
```

Comment:
This type was wrong, the spread goes into a `<span>` and _some_ attributes are piped into `ListboxButton`